examples/synology/nebula.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+SCRIPT="PATH_TO_CONFIG" # example /volume1/nebula/config.yml
+SUBNET="YOUR SUBNET IP" # example 10.1.0.0/24
+
+PIDFILE="PATH_TO_PID" # example - /volume1/nebula/nebula.pid
+LOGFILE="PATH_TO_LOG" # example - /volume1/nebula/nebula.log
+
+status() {
+  if [ -f $PIDFILE ]; then
+    echo 'Service running' >&2
+    return 1
+  fi
+}
+
+start() {
+  if [ -f $PIDFILE ] && kill -0 $(cat $PIDFILE); then
+    echo 'Service already running' >&2
+    return 1
+  fi
+  printf 'Starting nebula service...' >&2
+  "$SCRIPT" &> "$LOGFILE" & echo $! > "$PIDFILE"
+  sleep 5
+
+  NEXT_WAIT_TIME=0
+  until [ $NEXT_WAIT_TIME -eq 120 ] || [ ! -z "$(ifconfig | grep nebula1)" ]; do
```

Comment:
`nebula1` here assumes that we're using the example config. The default value might be `tun<x>`, or may have been defined in the users own config file. It's probably worth making this one of the configurable values at the top of the script.